### ISSUE TITLE
Copyright

### ISSUE THREAD

**Issue by 1010todd** [Hecter94/EndlessSky-PluginAssets#1]
As noted by MidnightPlugins on Discord we should also list the creator of these assets so anyone who used them can properly attribute the original creator. I think we can just follow the same format as [copyright file](https://github.com/endless-sky/endless-sky/blob/master/copyright) on the Endless Sky repo.

**Comment by Hecter94** [comment]
I agree that this is something that is going to need to be done, however given the sheer number of images, I believe this is going to need to be a long-term goal that gets worked on over the course of the repo's lifespan.

As is, I would first like to focus on getting both the plugin archive and asset repo into a workable and usable state, starting with the updating of any old plugins, and then focus on categorization, documentation and attribution. 

Of course, if more people are willing to help with this project as a whole, we may be able to start documenting sooner, but for now I think we should keep this on the backburner.
